### PR TITLE
Add a Field's initial value to props.meta.initial

### PR DESCRIPTION
**Pull request by bmv437**
This PR passes the initial value for a field down to the Field component as `props.meta.initial`

This is useful for having to compare the current value to the initial value to display a pending change. In my specific case, this enables checking a number never goes below the initial value, but can be increased and decreased to any value >= the initial value.

### CODE REVIEW

File: src/createFieldProps.js [hunk 1]
```diff
@@ -88,6 +89,7 @@ const createFieldProps = ({ getIn, toJS }, name,
       dispatch,
       error,
       form,
+      initial: initial,
```
Comment (idrumgood) [inline src/createFieldProps.js]: any reason for not just `initial`?

**Comment by bmv437** [comment]
@gustavohenke Does this feature look alright? Can we get it merged? Thanks for your efforts on redux-form!

**Comment by erikras** [comment]
Looks great!

**Comment by erikras** [comment]
Published as [`v6.7.0`](https://github.com/erikras/redux-form/releases/tag/v6.7.0).

**Pull request merged**